docs/source/configuring.md

```diff
@@ -265,6 +265,14 @@ If you use a version control system besides Git, you can use the environment var
 
 Currently, only Git is fully supported by Apollo Studio.
 
+## Bypass SSL/TLS Validation
```

Comment:
nit: I think it's still worth saying both TLS & SSL but I'd say put TLS first.   (TLS is really what it is meant to be called these days, and it supersedes the _long_ deprecated SSL.)

```suggestion
## Bypass TLS/SSL Validation
```